hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/resolver/backend/JdbcBackendResolver.java

```diff
@@ -79,7 +87,15 @@
     }
     entityBuilder.putAttributes(
         RawSpanConstants.getValue(Sql.SQL_DB_TYPE), createAttributeValue(dbType));
-    return Optional.of(entityBuilder.build());
+
+    Map<String, AttributeValue> enrichedAttributes = new HashMap<>();
+    String jdbcOperation =
+        SpanAttributeUtils.getFirstAvailableStringAttribute(
+            event, DbSemanticConventionUtils.getAttributeKeysForDbOperation());
+    if (jdbcOperation != null) {
```

Comment:
FYI: For now, no action, we will see if we have to parse `db.statement` using regex for open tracing convention as it doesn't have `db.operation`.